recipes/prgenv-gnu/25.6/gh200/extra/reframe.yaml

```diff
@@ -0,0 +1,14 @@
+default:
+  features:
+    - cuda
+    - mpi
+    - nccl-tests
+    - openmp
+    - osu-micro-benchmarks
+    - prgenv
```

Comment:
Note the newly added `prgenv` feature here. See https://github.com/eth-cscs/alps-uenv/pull/243 for more details.